src/autocomplete.css

```diff
@@ -29,6 +29,14 @@
   outline: 3px solid #ffbf47;
 }
 
+.autocomplete__input.autocomplete__input--show-all-values {
```

Comment:
Presumably both `.button--default` and `button--primary` could extend `.button` if we wanted.